web/wp-content/themes/dctx-v2/inc/acf.php

```diff
@@ -0,0 +1,449 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-b0ejyx.php, but found phpcs-scan-b0ejYX.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).